.github/workflows/weekly-resources-clean.yml

```
@@ -18,9 +18,25 @@ on:
     - cron: "0 0 * * SAT" # Run every saturday at midnight
   workflow_dispatch:
 
+env:
+  REGION_1: us-east-2
+  REGION_2: us-west-2
 jobs:
+  get-resources-cleaning-region:
+    runs-on: ubuntu-latest
+    outputs:
+      region-matrix: ${{ steps.set-matrix.outputs.region-matrix }}
+    steps:
+      - name: Get Region to clean SSM package
+        id: set-matrix
+        run: echo "::set-output name=region-matrix::{\"include\":[{\"region\":\"${{ env.REGION_1 }}\"},{\"region\":\"${{ env.REGION_2 }}\"}]}"
+
   clean-soaking-resources:
+    needs: [ get-resources-cleaning-region ]
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix: ${{fromJson(needs.get-resources-cleaning-region.outputs.region-matrix)}}
```

The environment variables aren't being used anywhere else. Keep it simple.
```suggestion
      matrix:
        region: [us-east-2, us-west-2]
```

---

Thought it the same as first but yeah, let's just keep it simple for now.